static/js/settings_account.js

```diff
@@ -100,6 +100,17 @@ exports.add_custom_profile_fields_to_settings = function () {
         });
         $("#account-settings .custom-profile-fields-form").append(html);
     });
+
+    $(".custom_user_field_value").on('change', function (e) {
+        var fields = [];
+        var value = $(this).val();
+        fields.push({id: parseInt($(e.target).closest('.custom_user_field').attr("data-field-id"), 10),
+                     value: value});
+        var spinner = $("#custom-field-status").expectOne();
+        loading.make_indicator(spinner, {text: 'Saving ...'});
+        settings_ui.do_settings_change(channel.patch, "/json/users/me/profile_data",
+                                       {data: JSON.stringify(fields)}, spinner);
+    });
```

Comment:
I don't think this is ideal, because it's not obvious we won't end up double-adding handlers.

I'd prefer to instead just move this to an `initialize` method that's called unconditionally on page load, and make the jquery selector be the  `$('#settings_overlay').on('change', '.custom_user_field_value', function()` style.